pkg/kubelet/cm/dra/plugin/noderesources.go

```diff
@@ -259,9 +260,20 @@ func (c *nodeResourcesController) run(ctx context.Context) {
 	// because eventually these list calls succeed.
 	// TODO (https://github.com/kubernetes/kubernetes/issues/123691): can we avoid these error log entries? Perhaps wait here?
 
+	var node *v1.Node
+	var err error
+	for {
+		node, err = c.getNode()
+		if err == nil {
+			break
+		}
+		logger.V(5).Error(err, "getting Node object failed, waiting")
+		time.Sleep(time.Second)
+	}
```

Comment:
@pohly This might also fix https://github.com/kubernetes/kubernetes/issues/123692
At least I don't see `resourceslices.resource.k8s.io is forbidden` in the kubelet logs in my environment.